streams_map_legacy.go

```diff
@@ -185,6 +185,14 @@ func (m *streamsMapLegacy) OpenStreamSync() (Stream, error) {
 	}
 }
 
+func (m *streamsMapLegacy) OpenUniStream() (SendStream, error) {
+	return nil, errors.New("gQUIC doesn't support unidirectional streams")
```

Comment:
Slight preference for making this a panic, as it shows a severe misusage of our library.